src/meta_memcache/base/cache_pool.py

```diff
@@ -418,6 +417,6 @@ def delta_initialize_and_get(
         result = self.meta_arithmetic(
             key=key, flags=flags, int_flags=int_flags, token_flags=token_flags
         )
-        if isinstance(result, Value):
+        if isinstance(result, Value) and result.value is not None:
```

Comment:
Same here, that is why I was suggesting to maybe move the conversion a level up, and make this return a `IntValue`. Either move the logic to the parent class, or repeat here the same isnumeric check?

Reply:
I've repeated it for now, it looks like these checks are specific to these methods